app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -146,7 +146,7 @@ def update_content_counts!
             end
           end
           new_content_counts[:content_view_versions][repo.content_view_version_id] ||= { repositories: {}, cv_version_content_counts: {}}
-          new_content_counts[:content_view_versions][repo.content_view_version_id][:repositories][repo.id] = translated_counts
+          new_content_counts[:content_view_versions][repo.content_view_version_id][:repositories][repo.content_view_version.archived_repos.find_by(library_instance_id: repo.library_instance_id)&.id] = translated_counts
```

Comment:
What do you think about adding a comment somewhere saying that the repos returned are only the archived ones?